### PR TITLE
Issue 68/configurable components sorting

### PR DESCRIPTION
**Pull request by naomijub**
Addresses issue #68

### CODE REVIEW

**Comment by naomijub** [comment]
@rewin123 this solution is cleaner and has reflect

File: src/editor/ui/inspector/mod.rs [hunk 1]
```diff
@@ -35,6 +39,9 @@ impl Plugin for SpaceInspectorPlugin {
     fn build(&self, app: &mut App) {
         app.init_resource::<InspectState>();
         app.init_resource::<FilterComponentState>();
+        app.init_resource::<ComponentsOrder>();
+        app.editor_component_priority::<Name>(0);
+        app.editor_component_priority::<Transform>(1);
```
Comment (naomijub) [inline src/editor/ui/inspector/mod.rs:44]: this section might live in another place, but for now seems good

**Comment by rewin123** [comment]
Yep. Its nice and clean now:)

**Comment by rewin123** [comment]
I tested. All works. Thanks!

**Pull request merged**